common/flatpak-installation.c

```diff
@@ -2663,3 +2663,30 @@ flatpak_installation_prune_local_repo (FlatpakInstallation *self,
 
   return flatpak_dir_prune (dir, cancellable, error);
 }
+
+/**
+ * flatpak_installation_run_triggers
```

Comment:
Missing a colon after the function name here: I don't know if gtk-doc cares.

Reply:
gtk-doc does care.

Reply:
(This was a copy paste from @smspillaz preceding function! Which I have now also fixed.)